src/app/components/TransactionsTable/index.tsx

```diff
@@ -37,13 +37,15 @@ export default function TransactionsTable({
   }
 
   return (
-    <div className="mt-4">
+    <div>
       {loading ? (
         <div className="w-full flex flex-col items-center">
           <Loading />
         </div>
-      ) : !transactions?.length && noResultMsg ? (
-        <p className="text-gray-500 dark:text-neutral-400">{noResultMsg}</p>
+      ) : !transactions?.length ? (
+        <p className="text-gray-500 dark:text-neutral-400">
+          {t("no_transactions")}
```

Comment:
we should remind ourself to always show something positive and maybe a call to action in such cases. 

empty states is the first thing the users sees and always somewhat sad.

Reply:
I agree, but out of scope for this PR.